docs/docs/graphql.md

```diff
@@ -1506,16 +1513,34 @@ The following is some basic setup information, but please consult [Docusaurus](h
 2. Create a `docs` directory at the root of your project
 
 ```terminal
-mcd docs
-`
+mkdir docs
+```
+
+
+3. Install Docusaurus
 
-3. Install the plugin
+```terminal
+npx create-docusaurus@latest docs classic
+```
+
+4. Install the GraphQL Generators Plugin
 
 ```terminal
-yarn add @edno/docusaurus2-graphql-doc-generator graphql
+yarn workspace docs add @edno/docusaurus2-graphql-doc-generator graphql
+```
+
+5. Create a Directory for your GraphQL APi generated documentation to reside in with the Docusaurus directory `/docs` structure
+
+```terminal
+// Change into the "docs" workspace
+
+cd docs
+
+// you should have the "docs" directory and within that a "graphql-api" directory
+mkdir docs/graphql-api
```

Comment:
This folder also exists after running the `npx` command with the following files: 

> intro.md
> tutorial-basics/
> tutorial-extras/

Reply:
Also, it would be good to explicitly describe the final folder path that needs to be created, because someone will think the path should be `/project/docs/graphql-api` not `/project/docs/docs/graphql-api`

Reply:
Changed to  Ensure a Directory for your GraphQL APi generated documentation resides